src/redux/initialState.js

```diff
@@ -970,7 +970,36 @@ const initialState = {
       newFurniture: true,
       hotDeal: true,
     },
+    {
+      id: 'Jone-Dove',
+      name: 'Jone Dove',
+      category: 'Furniture client',
+      text:
+        'There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which dont look even slightly believable.',
+      image: 'https://collectionimages.npg.org.uk/large/mw192616/John-Dove.jpg',
+      newFurniture: true,
+    },
+    {
+      id: 'Bob-Dylan',
+      name: 'Bob Dylan',
+      category: 'Furniture client',
+      text:
+        'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters',
+      image:
+        'https://bi.im-g.pl/im/1/11556/z11556131V,Bob-Dylan--portret-z-lat-60-.jpg',
+      newFurniture: true,
+    },
+    {
+      id: 'Marlon-Brando',
+      name: 'Marlon Brando',
+      category: 'Furniture client',
+      text:
+        'There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which dont look even slightly believable.',
+      image: 'https://fwcdn.pl/ppo/01/40/140/449871.2.jpg',
+      newFurniture: true,
+    },
```

Comment:
Dlaczego usunąłe `feedbacks` i dodałeś te wpisy do listy produktów? To są dwa odrębne rodzaje danych. USuń to proszę z tych produktów, W twoim zadaniu nie dotykasz produktów, tylko pokazuje opinie klientów